### PR TITLE
EG-1441-Updated release files

### PR DESCRIPTION
**Pull request by JessicaSavvy**

### CODE REVIEW

File: content/en/docs/cenm/1.1/release-notes.md [hunk 1]
```diff
@@ -17,14 +17,24 @@ title: Release notes
...
-* Bug fix: Identity Manager upgrade from CENM 0.4 causes JIRA Workflow Plugin to stop the existing tickets in status 'New' or
-'In Progress' from being progressed.
+* CENM 1.1.3 introduces fixes to known issues in CENM 1.1.
+
+Fixed issues
+* Identity Manager upgrade from CENM 0.4:
+  * causes JIRA Workflow Plugin to stop existing tickets in 'New' or 'In Progress' status from being progressed
+  * prevents rejected records from being cleared by `workflow_*`.
+* When multiple users are configured to use the Signing Service, the service authenticates all the credentials before checking whether the threshold is reached, which would result in multiple authentications for every user.
+* The process of creating and signing the CRL fails when upgrading from 0.4 if the existing revoked certificates lacked a revocation reason.
```
Comment (JayFarei) [inline content/en/docs/cenm/1.1/release-notes.md:29]: in the line just above we refer to CENM 0.4 we might want to do the same here?

File: content/en/docs/cenm/1.2/release-notes.md [hunk 2]
```diff
@@ -15,6 +15,16 @@ title: Release notes
...
 
+## Release 1.2.2
+
+CENM 1.2.2 introduces fixes to known issues in CENM 1.2.
+
+Fixed issues
+
+* Generating a certificate hierarchy using the PKI tool and a supported version of Gemalto HSM fails to generate keys.
+* Using `csr_token` as part of a node registration causes the registration to fail when the Identity Manager is set up to use a supported version of Oracle DB.
+* Creating and signing the CRL fails when upgrading from 0.4 if the existing revoked certificates lacked a revocation reason.
```
Comment (JayFarei) [inline content/en/docs/cenm/1.2/release-notes.md:26]: same as above - should this be CENM 0.4

File: content/en/docs/cenm/1.2/release-notes.md [hunk 3]
```diff
@@ -106,15 +116,24 @@ until the request is REJECTED or APPROVED. This means the external system needs
...
-* Bug fix: Identity Manager upgrade from CENM 0.4 causes JIRA Workflow Plugin to stop processing the existing tickets in status 'New' or 'In Progress'.
+CENM 1.1.3 introduces fixes to known issues in CENM 1.1.
+
+Fixed issues
+
+* Identity Manager upgrade from CENM 0.4:
+  * causes JIRA Workflow Plugin to stop existing tickets in 'New' or 'In Progress' status from being progressed
+  * prevents rejected records from being cleared by `workflow_*`.
+* When multiple users are configured to use the Signing Service, the service authenticates all the credentials before checking whether the threshold is reached, which would result in multiple authentications for every user.
+* The process of creating and signing the CRL fails when upgrading from 0.4 if the existing revoked certificates lacked a revocation reason.
```
Comment (JayFarei) [inline content/en/docs/cenm/1.2/release-notes.md:129]: same as above - CENM 0.4

File: content/en/docs/cenm/1.2/release-notes.md [hunk 4]
```diff
@@ -15,6 +15,16 @@ title: Release notes
 
 # Release notes
 
+## Release 1.2.2
+
+CENM 1.2.2 introduces fixes to known issues in CENM 1.2.
+
+Fixed issues
+
+* Generating a certificate hierarchy using the PKI tool and a supported version of Gemalto HSM fails to generate keys.
```
Comment (rnicoll) [inline content/en/docs/cenm/1.2/release-notes.md:24]: I think this was rather less dramatic a failure... @nkovacsx

**Comment by davidrapacchiale** [comment]
Looks good, @JessicaSavvy

**Comment by JessicaSavvy** [comment]
> Looks good, @JessicaSavvy

Thanks David, Ross has just spotted something about Gemalto which we might need to review/update.

File: content/en/docs/cenm/1.2/release-notes.md (hunk 4 above)
Comment (rnicoll) [reply content/en/docs/cenm/1.2/release-notes.md:24]: Confirming with Gabe but think this is a miscommunication and should be removed.

Comment (JayFarei) [reply content/en/docs/cenm/1.2/release-notes.md:24]: @rnicoll sounds like you covered it in the comment of this ticket: https://r3-cev.atlassian.net/browse/ENM-1160 // extra debug line vs an actual issue present in the patch - correct? if that is the case we should not mention this.

Comment (JayFarei) [reply content/en/docs/cenm/1.2/release-notes.md:24]: reporting a conversation off line - this was by mistake flagged in 1.2.2 fix version patch - the extent of the fix was indeed a debug line

**Comment by rnicoll** [comment]
LGTM

**Pull request merged**